File: 2 D Array/Matrix Traverse/matrixTraverse.cpp
#include<iostream>
#include<cstdio>
#define maxSize 100

using namespace std;

int main()
{
    int matrix[maxSize][maxSize], rowSize, columnSize;

    // Scanning the size of row of the matrix
    cout << "Enter the size of row of the matrix: ";
    cin >> rowSize;

    // Scanning the size of the column of matrix
    cout << "Enter the size of the row of matrix: ";
    cin >> columnSize;

    // Scanning the matrix
    for(int i=0; i<rowSize; i++)
    {
        for(int j=0; j<columnSize; j++)
        {
            printf("Element [%d][%d]: ",i,j);
            cin >> matrix[i][j];            
        }
    }       

    // Printing the matrix
    cout << endl << endl;
    cout << "Matrix:" << endl;
    for(int i=0; i<rowSize; i++)
    {
        for(int j=0; j<columnSize; j++)
        {
            cout << matrix[i][j] << " ";
        }
        cout << endl;
    }

    return 0;
}

File: 2 D Array/Matrix Traverse/matrixTraverse.py
# scanning the number of rows and column
row_num = int(input("Enter the number of rows: "))
col_num = int(input("Enter the number of columns: "))

# defining the 2-D array with initial value is 0
two_d_array = [[0 for col in range(col_num)] for row in range(row_num)]

# scanning the elements of the array
for row in range(row_num):
    for col in range(col_num):
        two_d_array[row][col] = int(input(f"Element[{row}][{col}]: "))

# printing the 2-D array
print("2-D Array:")
print(two_d_array)
